examples/mysql-s3import/main.tf

```diff
@@ -0,0 +1,120 @@
+provider "aws" {
+  region = "us-west-2"  
+}
+
+######################################
+# Data sources to get VPC and subnets
+######################################
+data "aws_vpc" "default" {
+  default = true
+}
+
+data "aws_subnet_ids" "all" {
+  vpc_id = data.aws_vpc.default.id
+}
+
+##############################################
+# Vars that are being used in multiple places
+##############################################
+locals {
+  s3_import_bucket_name               = "mysql-s3-restore"
```

Comment:
Please make sure that examples are 100% self-containing with all dependencies provisioned. In this case, you can create s3 bucket, upload a tiny backup file to there, and then call the `aurora` module to verify that it will create resources correctly.

Also, you can use `depends_on` to make sure that required S3 resources are created before Aurora starts the creation.

This is probably the hardest part of this PR but it is very important. We have examples in all modules already.